src/components/quiz/QuizQuestions.scss

```diff
@@ -0,0 +1,60 @@
+@import "../../constants.scss";
+
+*{  box-sizing: border-box;
+  background-color:$secondary-light;
+}  
```

Comment:
Whoa!! Don't use the `*` selector, that styles every element! This is very likely to affect other people's CSS in a drastic way. `box-sizing: border-box` will change how everyone else has sized their elements!